src/ttsnake.c

```diff
@@ -540,8 +617,9 @@ int main ()
 {
   struct sigaction act;
   int rs;
+  int nscenes;
   pthread_t pthread;
-  scene_t* intro_scene = malloc(sizeof(*intro_scene) * N_INTRO_SCENES);
+  scene_t* intro_scene;
   scene_t* game_scene = malloc(sizeof(*game_scene) * N_GAME_SCENES);
```

Comment:
This is probably not that important here, but when it comes to  good practices,  checking all library functions which make syscalls is often recommended. In this case, it would be cleaner to call malloc in the function body, rather than in the initialization (and user `sysfatal` from `utils.h`).

Reply:
Fixed